ddl/db_integration_test.go

```diff
@@ -1338,6 +1338,9 @@ func TestAddIndexAfterAddColumn(t *testing.T) {
 	tk.MustExec("insert into test_add_index_after_add_col values(1, 2),(2,2)")
 	tk.MustExec("alter table test_add_index_after_add_col add column c int not null default '0'")
 	sql := "alter table test_add_index_after_add_col add unique index cc(c) "
+	limit := variable.GetDDLErrorCountLimit()
+	variable.SetDDLErrorCountLimit(3)
+	defer variable.SetDDLErrorCountLimit(limit)
```

Comment:
Why do we need this? Normally, the error should be able to return when it occurs at the first time.